easy-ui-react/src/TabNav/TabNav.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React, { ComponentProps, useState } from "react";
+import { TabNav, TabNavProps } from "./TabNav";
+
+type Story = StoryObj<typeof TabNav>;
+
+const meta: Meta<typeof TabNav> = {
+  title: "Components/TabNav",
+  component: TabNav,
+};
+
+export default meta;
+
+export const Default: Story = {
+  render: DefaultTemplate.bind({}),
+};
+
+export const CustomLink: Story = {
+  render: CustomLinkTemplate.bind({}),
+};
+
+export const Responsive: Story = {
+  render: () => (
+    <div style={{ maxWidth: 360 }}>
+      <CustomLinkTemplate />
+    </div>
+  ),
+};
+
+const tabs = [
+  ["Billing", "billing"],
+  ["Members", "members"],
+  ["API Keys", "api-keys"],
+  ["Branded Tracker", "branded-tracker"],
+  ["Shipping Settings", "shipping-settings"],
+];
+
+function DefaultTemplate(args: TabNavProps) {
+  return (
+    <TabNav aria-label="Account" {...args}>
+      {tabs.map(([label, location], i) => (
+        <TabNav.Item
+          key={location}
+          href={href(location)}
+          isCurrentPage={tab() === location || (!tab() && i === 0)}
+        >
+          {label}
+        </TabNav.Item>
+      ))}
+    </TabNav>
+  );
+}
+
+function CustomLinkTemplate(args: Partial<TabNavProps>) {
+  const [page, setPage] = useState("billing");
+  return (
+    <TabNav aria-label="Account" {...args}>
+      {tabs.map(([label, location]) => (
+        <TabNav.Item
+          key={location}
+          as={FakeClientSideRouterLink}
```

Comment:
This is appearing as ` as={function Jo(){}}` in the `Show code` view in Storybook

Reply:
yeah this is annoying. i looked into trying to fix this and tried various adjustments like adding a `displayName` to the component and couldn't get react docgen to pick up anything other than an anonymous function. seems like it may be a limitation of react-docgen at the moment. i'll keep an eye out on how to fix this though